src/Composer/Package/BasePackage.php

```diff
@@ -23,11 +23,11 @@
 abstract class BasePackage implements PackageInterface
 {
     public static $supportedLinkTypes = array(
```

Comment:
I guess this could be typed like

```suggestion
    /**
     * @phpstan-var array<string, array<description: string, method: Link::TYPE_*>>
     */
    public static $supportedLinkTypes = array(
```

Reply:
thanks, I added this